avm/res/compute/virtual-machine-scale-set/extension/version.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We don't version child modules in AVM. Please remove

Reply:
done